snap/snapcraft.yaml

```diff
@@ -26,7 +26,6 @@ parts:
     parse-info: [usr/share/appdata/mailspring.appdata.xml]
     stage-packages:
       - libxkbfile1
-      - libnss3
```

Comment:
I think the snap linter was also saying this was unused -- let's try removing it and I'll put it back if anyone has issues with the snap :-)